lib/sudo-common/src/error.rs

```diff
@@ -1,10 +1,20 @@
-#[derive(Debug)]
+use sudo_pam::PamError;
+use thiserror::Error;
```

Comment:
I propose to not use a dep on thiserror for now. Implementing this ourselves does not introduce to much code - see my PR.